components/layout/dom_traversal.rs

```diff
@@ -219,6 +221,18 @@ fn traverse_children_of<'dom>(
         } else {
             handler.handle_text(&info, node_text_content);
         }
+    } else if parent_element.is_text_control_inner_editor() {
+        // We are using parent's style for caret and selection color.
+        // FIXME: Once textarea and input element's shadow tree is supported, this would replace the if condition above.
+        let info = NodeAndStyleInfo::new(
```

Comment:
I don't understand this. Shouldn't `child` style be the same as `parent_element`?